### DISCUSSION TITLE
Public semi-automated benchmarks

### DISCUSSION THREAD

**Discussion by abelsiqueira** [JuliaSmoothOptimizers/Organization#40] [Ideas]
1. It would be nice to have some automated generation of benchmarks for all our solvers.
2. It would be nice to have some benchmarks comparing to other packages. I would be better if an interested third-party would maintain this, kind like the AC-OPF benchmarks.

**Comment by RoyiAvital** [comment]
It would be great to have some comparison with MATLAB.

For instance [`RipQP.jl`](https://github.com/JuliaSmoothOptimizers/RipQP.jl) vs. MATLAB's [`lsqlin()`](https://www.mathworks.com/help/optim/ug/lsqlin.html).

**Comment by RoyiAvital** [comment]
By the way, talking on `RipQP.jl`, You may share it in https://discourse.julialang.org/t/93394.  
As it seems many are looking for an alternative for MATLAB's `lsqlin()`. 

I personally looked for and thought that @dpo must have something related in his organization.

**Reply by dpo** [reply]
That would be useful indeed (any volunteers?) if we compare RipQP to lsqlin's interior-point option. Looking at the documentation quickly reveals that they pass the least-squares problem wholesale to the solver: https://www.mathworks.com/help/optim/ug/least-squares-model-fitting-algorithms.html#buc5ri4. That's not what I would recommend.

**Reply by RoyiAvital** [reply]
@dpo , If you set some interesting cases for the matrix I think I will be able to compare them both on the same computer.
